conformance_tests/sysman/test_sysman_init/src/test_init_sysman.cpp

```diff
@@ -24,4 +244,401 @@ TEST(SysmanInitTests,
   ASSERT_GT(pCount, 0);
 }
 
+TEST(SysmanInitTests, GivenSysmanInitializedThenCallingCoreInitSucceeds) {
```

Comment:
SysmanInitialized , CoreInit seem internal terms.
Can we use zesINit, zeInit instead ?

Reply:
Updated as per the suggestion